core/src/Kokkos_Macros.hpp

```diff
@@ -182,19 +186,16 @@
 //----------------------------------------------------------------------------
 // Intel compiler macros
 
-#if defined(KOKKOS_COMPILER_INTEL)
-// FIXME_ICPX
-#if !defined(__INTEL_LLVM_COMPILER)
+// FIXME_INTEL_LLVM
+#if defined(KOKKOS_COMPILER_INTEL_LLVM)
```

Comment:
The sense of the test got inverted here, I think? It had been classic `icpc` only. I think the `FIXME` comment was to see whether we should be enabling these pragmas for that as well.

Reply:
Fixed.